### PR TITLE
Add `DbHelper::class`.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: src/MessageSource.php [hunk 1]
```diff
@@ -17,6 +17,11 @@
 use function array_key_exists;
 use function is_string;
 
+/**
+ * Allows using database as message source for `yiisoft/translator`.
+ *
+ * Database schema can be initialized, you can use the {@see Migration::ensureTable()} method for this.
```
Comment (samdark) [inline src/MessageSource.php:23]:
```suggestion
 * Use the {@see Migration::ensureTable()} to initialize database schema.
```

File: src/MessageSource.php [hunk 2]
```diff
@@ -25,14 +30,12 @@ final class MessageSource implements MessageReaderInterface, MessageWriterInterf
     private array $messages = [];
     private string $sourceMessageTable = '{{%source_message}}';
     private string $messageTable = '{{%message}}';
-    private int $cachingDuration = 3600;
 
     public function __construct(
         private ConnectionInterface $db,
         private CacheInterface|null $cache = null,
-        int $cacheDuration = 3600
+        private int $cachingDuration = 3600
```
Comment (samdark) [inline src/MessageSource.php:37]: Need an ability to specify table names similar to how it's done in cache PR.

**Review by darkdef — APPROVED** [review]

File: src/DbHelper.php [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+                $command,
+                $schema,
+                $tableRawNameSourceMessage,
+                $tableRawNameMessage
+            );
+
+            return;
+        }
+
+        self::ensureTableNotSqlite(
```
Comment (vjik) [inline src/DbHelper.php:52]: Suggest explicitly describe drivers that supported and throw exception on not supported drivers.

File: src/DbHelper.php [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+use Yiisoft\Db\Schema\SchemaInterface;
+
+final class DbHelper
+{
+    /**
+     * @throws Exception
+     * @throws InvalidConfigException
+     * @throws Throwable
+     */
+    public static function ensureTable(
```
Comment (samdark) [inline src/DbHelper.php:23]:
```suggestion
    public static function ensureTables(
```

File: src/DbHelper.php [hunk 5]
```diff
@@ -0,0 +1,335 @@
...
+            $tableRawNameMessage
+        );
+    }
+
+    /**
+     * @throws Exception
+     * @throws InvalidConfigException
+     * @throws Throwable
+     */
+    public static function dropTable(
```
Comment (samdark) [inline src/DbHelper.php:66]:
```suggestion
    public static function dropTables(
```

File: src/DbHelper.php (hunk 3 above)
Comment (vjik) [reply src/DbHelper.php:52]: > All drivers are supported, in this particular case, the only difference is `Sqlite`, and that's why I call them that way.

There is no guarantee that all are supported. For example, can created yet another driver, that requried specific code as for sqlite.

Comment (vjik) [reply src/DbHelper.php]: Class has specific code for `oci`, `sqlite`, `sqlsrv`. I suggest explicitly describe supported drivers and throw exception if driver not included in this list.

Comment (vjik) [reply src/DbHelper.php]: There is no guarantee that driver name always will be Mssql, Mysql, MariaDB, Oracle, Pgsql, Sqlite only.

**Review by samdark — APPROVED** [review]

**Pull request merged**